client/src/app/core/repositories/motions/motion-repository.service.ts

```diff
@@ -104,12 +104,32 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
             'category_weight',
             'created',
             'last_modified',
-            'sequential_number'
+            'sequential_number',
+            'workflow_id',
```

Comment:
Same as in the other PR: Do you have an advantage of having id fields declared here?